commit 7a343f4c7cfc68b8331529c7bbe966b4151e837f (refs/changes/73/1273/2)
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2017-01-18T11:27:45-08:00 (2 years, 9 months ago)
    
    AGENT-1055 cn-agent copy of apm missing AGENT-1040
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/apm.js b/lib/apm.js
index 9feb04a..f86c3de 100755
--- a/lib/apm.js
+++ b/lib/apm.js
@@ -1,3 +1,13 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright (c) 2017, Joyent, Inc.
+ */
+
 /*
  *
  * apm.js - Ain't a Package Manager
@@ -357,7 +367,7 @@ function installPackage(toInstall, cb) {
             runLifecycleScript(
                 'preinstall',
                 packageJson,
-                installeddir,
+                thingdir,
                 function (error) {
                     if (error) {
                         return callback(error);
